packages/discovery/src/index.ts

```diff
@@ -0,0 +1,23 @@
+// DNS Discovery
+export { PeerDiscoveryDns, wakuDnsDiscovery } from "./dns/dns_discovery.js";
+export { enrTree } from "./dns/constants.js";
+export { DnsNodeDiscovery } from "./dns/dns.js";
+
+// Peer Exchange Discovery
+export {
+  wakuPeerExchange,
```

Comment:
Wondering if we need these `waku*`. To me it seems we can avoid and just be straight - `peerExchangeDiscovery`, `peerCacheDiscovery`, etc

Reply:
Removed two exports, but we use reft of them somewhere or the other (mostly for different types of tests)

Please feel free to take a look and suggest if there's an unnecessary export remaining